Library/Formula/dependency-check.rb

```diff
@@ -25,6 +25,10 @@ def install
     (etc/"dependencycheck").mkpath
     corejar = libexec/"repo/org/owasp/dependency-check-core/#{version}/"\
       "dependency-check-core-#{version}.jar"
+    if File.exist? etc/"dependencycheck/dependencycheck.properties"
```

Comment:
Does upstream actually overwrite the configuration files if one already exists? That's fairly unusual behaviour.